gds_py/Dockerfile

```diff
@@ -57,7 +65,6 @@ RUN nbdime extensions --enable --user $NB_USER \
  && jupyter labextension update nbdime-jupyterlab
 
 # Clean up
-RUN conda clean -tipsy
```

Comment:
Moved to the end of the `conda` command to avoid an extra layer. I would join these into a single layer too.